<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PythonPanel</class>
 <widget class="QWidget" name="PythonPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QWidget, QScrollArea, .QFrame {
background-color: white;
}

QTabWidget:pane {
border: 0px; 
}

QTabBar::tab {
   color: #333;
border: 1px solid #000;
/*border-top-left-radius: 11px;
border-top-right-radius: 11px;*/
border-bottom: none;
padding: 2px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888);
min-width: 60px;
}

QTabBar::tab:hover {
    background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}

QTabBar::tab:selected {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
    stop: 0 #EFEFEF, stop: 1 #EEEEEE);
    color: #3c3c3c;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="HeaderFrame" name="headerFrame" native="true">
     <property name="expandable" stdset="0">
      <bool>false</bool>
     </property>
     <property name="title" stdset="0">
      <string>Python</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PythonTabWidget" name="tabWidget">
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane { 
     border-top: none;
 }</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <property name="toolTip">
       <string>A Python Read-Eval-Print-Loop to manipulate interactively the graphs opened in Tulip</string>
      </property>
      <attribute name="title">
       <string>REPL</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="graphComboTitle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;b&gt;Selected graph:&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TreeViewComboBox" name="graphCombo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>The selected graph will be bound to a Python global variable named &quot;graph&quot;</string>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToMinimumContentsLength</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="tlp::PythonShellWidget" name="pythonShellWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="toolTip">
       <string>Python output console for Tulip Python plugins</string>
      </property>
      <attribute name="title">
       <string>Output</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="consoleWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="clearButton">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/tulip/graphperspective/icons/16/edit-clear.png</normaloff>:/tulip/graphperspective/icons/16/edit-clear.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PythonTabWidget</class>
   <extends>QTabWidget</extends>
   <header location="global">tulip/PythonTabWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HeaderFrame</class>
   <extends>QWidget</extends>
   <header>HeaderFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tlp::PythonShellWidget</class>
   <extends>QWidget</extends>
   <header location="global">tulip/PythonShellWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TreeViewComboBox</class>
   <extends>QComboBox</extends>
   <header>tulip/TreeViewComboBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
